jenkins/agent-ctl.sh

```diff
@@ -75,12 +75,21 @@ while true; do
       FAKE=yes
       shift
       ;;
+    --previous)
```

Comment:
meh, I just want someplace to note the last version we were using. I think this is easier for someone reading the code to understand than `# previous --pr was jenkins-agent-20160613-2240` but would you prefer I do it that way?